app/web/polyfills.js

```diff
@@ -9,11 +9,6 @@ const browser = {
   shell: {
     openExternal: (url) => window.open(url, ''),
   },
-  remote: {
```

Comment:
how this change is related to the original PR?

Reply:
It is the browser version's polyfill equivalent of Electron's `remote` polyfill. Since the latter has been removed, the former can be removed, too.